Framework/Source/Graphics/RenderGraph/RenderPassesLibrary.cpp

```diff
@@ -55,22 +57,30 @@ namespace Falcor
         RenderPassLibrary::addRenderPassClass("ToneMapping", "Tone-Mapping", ToneMapping::deserialize);
         RenderPassLibrary::addRenderPassClass("FXAA", "FXAA", FXAA::deserialize);
         RenderPassLibrary::addRenderPassClass("SSAO", "Fast Approximate Anti-Aliasing", SSAO::deserialize);
-        RenderPassLibrary::addRenderPassClass("TAA", "Temporal Anti-Aliasing", TemporalAA::deserialize);
+        RenderPassLibrary::addRenderPassClass("TemporalAA", "Temporal Anti-Aliasing", TemporalAA::deserialize);
+        
+        RenderPassSerializer skyBoxSerializer;
+        skyBoxSerializer.addVariable<uint32_t>("sampleDesc.minFilter", static_cast<uint32_t>(Sampler::Filter::Linear));
+        skyBoxSerializer.addVariable<uint32_t>("sampleDesc.magFilter", static_cast<uint32_t>(Sampler::Filter::Linear));
+        skyBoxSerializer.addVariable<uint32_t>("sampleDesc.mipFilter", static_cast<uint32_t>(Sampler::Filter::Linear));
+        skyBoxSerializer.addVariable<bool>("loadAsSrgb", true);
+        
+        RenderPassLibrary::addRenderPassClass("SkyBox", "Sky Box pass", SkyBox::deserialize, skyBoxSerializer);
 
         return true;
     };
 
     static const bool b = addBuiltinPasses();
 
-    void RenderPassLibrary::addRenderPassClass(const char* className, const char* desc, CreateFunc func)
+    void RenderPassLibrary::addRenderPassClass(const char* className, const char* desc, CreateFunc func, RenderPassSerializer serializer)
```

Comment:
No need for the serializer here. When the users call  createRenderPass() they should pass a serializer object, otherwise a RenderPass should know how to create a valid object when no serializer is provided